acquia_lift.install

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/**
+ * @file
+ * Installation file for Acquia Lift module.
+ */
+
+use Drupal\acquia_lift\Service\Helper\SettingsHelper;
+
+/**
+ * Implements hook_uninstall().
+ *
+ * Remove cache directory if module is uninstalled.
+ */
+function acquia_lift_uninstall() {
+  // Purging all Acquia Lift settings and js libraries.
+  drupal_flush_all_caches();
```

Comment:
Yes you are right. I uninstalled the module without the uninstall hook and saw full page regen. Removing the uninstall hook.